# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/xjdic23/errno.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xjdclient.c.old	2003-03-30 20:04:13.000000000 +0100
+++ ./xjdclient.c	2003-03-30 20:05:12.000000000 +0100
@@ -36,6 +36,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define CVERBOSE 0
--- ./xjdserver.c.old	2003-03-30 20:04:22.000000000 +0100
+++ ./xjdserver.c	2003-03-30 20:05:31.000000000 +0100
@@ -28,6 +28,7 @@
 #include <netdb.h>
 #include <stdlib.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define SVERBOSE 0
